.github/workflows/main.yml

```diff
@@ -63,6 +63,7 @@ jobs:
         env:
           ENABLE_SEARCH: 'true'
           NODE_OPTIONS: '--max-old-space-size=6144'
+          GATSBY_EXPERIMENTAL_QUERY_CONCURRENCY: 32,
```

Comment:
For your new M2 max pro enterprise CPU :-)

Reply:
🤪

I wanted to try to max value first and see what happens 🙌

Reply:
In any case, with no cache now it took 7-8 min. Seems already a small improvement.